packages/cli/completions/wm.bash

```
@@ -146,6 +162,17 @@ _wm_completion() {
           COMPREPLY=( $(compgen -W "default project user" -- ${cur}) )
           return 0
           ;;
+        --config)
+          COMPREPLY=( $(compgen -f "${cur}") )
+          return 0
+          ;;
+        --tag|--mention|--property|--ref|--depends|--needs|--blocks|--id|--criteria|--file|--content-pattern|--contains|--command)
+          # These flags expect free-form values; fall back to default path/word completion
+          ;;
+        --limit|--page|--context|--after|--before)
+          COMPREPLY=( $(compgen -W "" -- ${cur}) )
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)